rsync-formula/pillar.example

```diff
@@ -25,6 +25,8 @@ rsync:
       hosts allow:
         - 2001:db8::1/128
         - 2001:db8:a::/64
+      # special case, specifying this exact line for any module will install a formula provided nameconvert.py
+      name converter: /usr/local/bin/nameconvert.py
```

Comment:
Should we also mention that the user would want to set `numeric ids: false`? And maybe point to the related documentation, like `MODULE PARAMETERS >  name converter, numeric ids`in `rsync.conf(5)`?